core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1399,6 +1398,14 @@ class EMConfig {
     @Cfg("Whether to skip failed SQL commands in the generated test files")
     var skipFailureSQLInTestFile = false
 
+    val defaultTreeDepth = 11
+
+    @Experimental
+    @Cfg("Maximum number of genes in mutations/queries to be evaluated;" +
+            "this is to avoid issues when dealing with huge graphs in GraphQL")
+    @Min(1.0)
+    var treeDepth = defaultTreeDepth
```

Comment:
when changing the name, also need to update the documentation :) ie, the comment `Maximum number of genes` inside `@Cfg`  is inconistent

Reply:
Done.